### PR TITLE
Bump the KCEF dependency version

### PR DESCRIPTION
**Pull request by SamCosta1**

### CODE REVIEW

File: sample/desktopApp/src/jvmMain/kotlin/main.kt [hunk 1]
```diff
@@ -32,7 +33,12 @@ fun main() =
                                 initialized = true
                             }
                         }
-                        release("jbr-release-17.0.10b1087.23")
+                        download(
+                            Download.Builder().github {
+                                release("jbr-release-17.0.10b1087.23")
```
Comment (DatL4g) [inline sample/desktopApp/src/jvmMain/kotlin/main.kt:38]: This version wont work anymore I guess.

The KCEF release tag is aligned to the date when jcef was updated.
In this case `20 April 2024`. (They updated some ID from `String` to `Long` if I remember correctly)

So probably [Release 17.0.10b1207.14](https://github.com/JetBrains/JetBrainsRuntime/releases/tag/jbr-release-17.0.10b1207.14) or [Release 17.0.11b1207.20](https://github.com/JetBrains/JetBrainsRuntime/releases/tag/jbr-release-17.0.11b1207.20) will work, but I haven't tested it.

Additionally a cleaner way that should work too would be:
```kotlin
download {
    release("<version>")
}
```

Comment (SamC-Apadmi) [reply sample/desktopApp/src/jvmMain/kotlin/main.kt:38]: Oh it must have been cached when I tested it, I'll try one of those

**Comment by SamC-Apadmi** [comment]
@DatL4g Sorry to push, I'm a bit blocked by this

**Comment by DatL4g** [comment]
@SamC-Apadmi even though I have repository access, I just review desktop request/changes and @KevinnZou approves and merges them

**Comment by SamC-Apadmi** [comment]
@KevinnZou Apologies I hadn't realised there was a klint job, I've fixed this now :) Should be ready to re-run

**Comment by SamC-Apadmi** [comment]
@KevinnZou Can this be merged?

**Comment by SamC-Apadmi** [comment]
❤️ ❤️ ❤️

**Comment by adden00** [comment]
so when are you planning to release version with this fix to maven?

**Comment by adden00** [comment]
thanks! it works

**Comment by DatL4g** [comment]
@KevinnZou lets bump it to `dev.datlag:kcef:2024.04.20.4`

This version supports loading jcef from the jdk, so apps build with the Jetbrains Jcef sdk won't need to download anything anymore.

**Pull request merged**